### PR TITLE
 RDoc-1312 Review security documentation 

### PR DESCRIPTION
**Pull request by iftahbe**

### CODE REVIEW

File: Documentation/4.1/Raven.Documentation.Pages/server/security/authentication/certificate-renewal-and-rotation.markdown [hunk 1]
```diff
@@ -14,7 +14,37 @@ You can also ignore these limits and replace the certificates immediately but be
 
 ## Replace the Cluster Certificate using the Studio
 
-This feature is under development and will be available in the Studio very soon. In the meantime please use the [RavenDB CLI command](../../../server/administration/cli#replaceclustercert).
+Access the certificate view and click on `Cluster certificate` -> `Replace cluster certificate`. Upload the PFX file and name it.
+
+This will start the certificate replacement process.
+
+If you encounter a problem during the replacement process, and there are no alerts in tne studio telling you what's wrong, please check the logs for errors and exceptions.
```
[the remark this answers is not in the document]
Comment (iftahbe) [reply Documentation/4.1/Raven.Documentation.Pages/server/security/authentication/certificate-renewal-and-rotation.markdown]: No, your are right, it doesn't make sense, I'll rephrase.

File: Documentation/4.1/Raven.Documentation.Pages/server/security/authentication/certificate-renewal-and-rotation.markdown [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+When using the Setup Wizard to obtain a Let's Encrypt certificate, you don't have to worry about this. Read about [Automatic Let's Encrypt Renewals in RavenDB](../../../server/security/authentication/lets-encrypt-certificates).
+
+If you provided your own certificate to RavenDB, it is **your responsibility** to renew it. 
+
+Once you have a new valid certificate for your server/cluster you need to make RavenDB use it instead of the currently loaded certificate. Replacing a certificate in the cluster is a distributed operation which requires all the nodes to confirm the replacement. The actual update will happen when all nodes of the cluster confirm the replacement or when there are 3 days left for expiration. 
+
+You can also ignore these limits and replace the certificates immediately but beware of this option. Nodes which didn't confirm the replacement, will not be able to re-join the cluster and will have to be setup manually. This means the new certificate will have to be placed manually in that node and [settings.json](../../configuration/configuration-options#json) will have to be edited, to contain the new certificate path. Then a server restart is required.
```
Comment (ayende) [inline Documentation/4.1/Raven.Documentation.Pages/server/security/authentication/certificate-renewal-and-rotation.markdown]: Server restart isn't required, you can overwrite the existing cert path which will cause us to reload the file within one hour or so

File: Documentation/4.1/Raven.Documentation.Pages/server/security/authentication/certificate-renewal-and-rotation.markdown [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+{DANGER The new certificate must contain all of the cluster domain names in the CN or ASN properties of the certificate. Otherwise you will get an authentication error because SSL/TLS requires the domain in the certificate to match with the actual domain being used. /}
+
+## Replace the Cluster Certificate using the Studio
+
+Access the certificate view and click on `Cluster certificate` -> `Replace cluster certificate`. Upload the PFX file and name it.
```
Comment (ayende) [inline Documentation/4.1/Raven.Documentation.Pages/server/security/authentication/certificate-renewal-and-rotation.markdown]: why do we need to name it?

Comment (iftahbe) [reply Documentation/4.1/Raven.Documentation.Pages/server/security/authentication/certificate-renewal-and-rotation.markdown]: Yes it's not necessary, we should also fix the studio view to remove the name field.

File: Documentation/4.1/Raven.Documentation.Pages/server/security/authentication/certificate-renewal-and-rotation.markdown (hunk 2 above)
Comment (iftahbe) [reply Documentation/4.1/Raven.Documentation.Pages/server/security/authentication/certificate-renewal-and-rotation.markdown]: yes, fixing

File: Documentation/4.0/Raven.Documentation.Pages/server/security/authentication/certificate-renewal-and-rotation.markdown [hunk 4]
```diff
@@ -8,13 +8,15 @@ If you provided your own certificate to RavenDB, it is **your responsibility** t
...
 
-You can also ignore these limits and replace the certificates immediately but beware of this option. Nodes which didn't confirm the replacement, will not be able to re-join the cluster and will have to be setup manually. This means the new certificate will have to be placed manually in that node and [settings.json](../../configuration/configuration-options#json) will have to be edited, to contain the new certificate path. Then a server restart is required.
+You can also ignore these limits and replace the certificates immediately but beware of this option. Nodes which didn't confirm the replacement, will not be able to re-join the cluster and will have to be setup manually. This means the new certificate will have to be placed manually in that node. 
+
+To manually replace the server certificate you can either edit [settings.json](../../configuration/configuration-options#json) with a new certificate path and restart the server or you can overwrite the existing certificate file and the server will pick it up within one hour without requiring a restart.
```
Comment (arekpalinski) [inline Documentation/4.0/Raven.Documentation.Pages/server/security/authentication/certificate-renewal-and-rotation.markdown]: "one hour" - do we have config option for that?

Comment (ayende) [reply Documentation/4.0/Raven.Documentation.Pages/server/security/authentication/certificate-renewal-and-rotation.markdown]: No, hard coded in `RefreshClusterCertificate`

**Pull request merged**